lib/CL/pocl_cl.h

```diff
@@ -996,8 +996,9 @@ struct _cl_mem {
      the shared system mem allocation */
   cl_device_id shared_mem_allocation_owner;
   /* device where this mem obj resides */
-  volatile cl_device_id owning_device;
-  /* A linked list of regions of the buffer mapped to the 
+  cl_device_id owning_device;
```

Comment:
The issue of buffer ownership and migrations is not touched at all in this PR. The meaning of owning device hasn't changed since the original commit by Ville that introduced it.